Applications/Games/Subnautica/Steam/script.js

```diff
@@ -0,0 +1,28 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+include(["Engines", "Wine", "Verbs", "vcrun2013"]);
+include(["Engines", "Wine", "Verbs", "vcrun2008"]);
+include(["Engines", "Wine", "Verbs", "d3dx9"]);
+include(["Engines", "Wine", "Verbs", "xact"]);
+include(["Engines", "Wine", "Verbs", "corefonts"]);
+
+new SteamScript()
+	.name("Subnautica")
+	.editor("Unknown Worlds Entertainment")
+	.author("Zemogiter")
+	.applicationHomepage("https://unknownworlds.com/subnautica/")
+	.wineDistribution("upstream")
+	.wineVersion(3.1)
+	.wineArchitecture("amd64")
+	.appId(264710)
+	.preInstall(function(wine, wizard) {
+        	wine.vcrun2013();
+        	wine.corefonts();
+		wine.vcrun2008();
+		wine.d3dx9();
```

Comment:
Why install d3dx9 when the game is using d3dx11 ?

And I don't think csmt is enabled by default in wine 3.1, so you might consider adding `wine.enableCSMT();`

Reply:
If you look at AppDB's page for Steam, d3dx9_42 is listed there as a recommended component. Since there is no d3dx9_42 verb, I've decided to use this. The user have to add -force-glcore or -force-opengl to the game's Launch Settings inside Steam. Otherwise textures for pretty much everything will be invisible. I dont know how to add the message about this to script. 
Also this script needs a line to add "UseTakeFocus" string value to HKEY_CURRENT_USER\Software\Wine\X11 Driver and set it to "N". Otherwise the game will crash when quiting it. Again I don't know how to do it in javascript and I haven't seen other game scripts doing this.
Isn't CSMT only in staging versions of wine?

Reply:
To force OpenGL add:
```Javascript
.executable("Steam.exe", ["-silent", "-applaunch", 264710, "-no-ces-sandbox", "-force-opengl"]) 
```
If you force OpenGL, it doesn't make any sense to me to install DirectX 9.

For the "UseTakeFocus", it's probably the best to add a new function to the [Wine script](https://github.com/PhoenicisOrg/Scripts/blob/master/Engines/Wine/Engine/Object/script.js). Compare e.g. `Wine.prototype.nativeApplication`.

As far as I remember CSMT should be also in stable since 3.0 or so (not quite sure, though).